plugin/pick.vim

```diff
@@ -68,3 +72,15 @@ function! s:BufferListCommand()
   let buffers = map(bufnrs, 'bufname(v:val)')
   return 'echo "' . join(buffers, "\n") . '"'
 endfunction
+
+function! s:TagCommand()
+  let l:tag_files = []
+
+  for tfile in split(&tags, ",")
+    if filereadable(expand(tfile)) == 1
```

Comment:
I wonder if race conditions will actually be a concern in a vim plugin.

But interesting idea, I like it alot! Fixed in 0b1472d